src/layouts/AccessWalletLayout/components/HardwareModal/HardwareModal.vue

```diff
@@ -130,14 +134,32 @@ export default {
       this.items.forEach(item => {
         const u2fhw = ['secalot', 'ledger', 'bitbox'];
         const inMobile = ['secalot', 'keepkey'];
+        const webUsb = ['keepkey'];
+
+        if (webUsb.includes(item.name)) {
+          const disable =
```

Comment:
I think you meant:

```
          const disable =
            window.location.protocol !== 'https:' ||
            (!window || !window.navigator || !window.navigator.usb);
```

I.e

1. webusb doesn't work without https
1. webusb doesn't work without window.navigator.usb, which doesn't work if there's no window.navigator, which doesn't work if there's no window.

Reply:
@gamalielhere sorry I missed this by a couple of minutes

Reply:
Thanks for the correction! Included the fix on #711